extensions/schema-compare/src/utils.ts

```diff
@@ -127,3 +127,8 @@ export async function verifyConnectionAndGetOwnerUri(endpoint: mssql.SchemaCompa
 export function getRootPath(): string {
 	return vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri.fsPath : os.homedir();
 }
+
+export function isSystemDatabase(name: string): boolean {
+	const systemDbs = new Set<string>(['master', 'model', 'msdb', 'tempdb']);
```

Comment:
https://docs.microsoft.com/en-us/sql/relational-databases/databases/system-databases?view=sql-server-ver15

Reply:
These are the 4 system dbs that can be connected to. Resource database can't be connected to.